src/spyglass/common/populate_all_common.py

```diff
@@ -9,6 +9,7 @@
 from .common_nwbfile import Nwbfile
 from .common_session import Session
 from .common_task import TaskEpoch
+from spyglass.spikesorting.merge import SpikeSortingOutput
```

Comment:
This import has issues.
Results in  a timeout when importing any part of spyglass since this file is imported as part of `spyglass.common.__init__.py`.  I'll dig in more and try to find where

Reply:
fixed